server/auth/types/openid/openid_auth.ts

```diff
@@ -106,6 +109,32 @@ export class OpenIdAuthentication extends AuthenticationType {
     }
   }
 
+  private createWreckClient(): typeof wreck {
+    const wreckHttpsOption: WreckHttpsOptions = {};
+    if (this.config.openid?.root_ca) {
+      wreckHttpsOption.ca = [fs.readFileSync(this.config.openid.root_ca)];
+    }
+    if (this.config.openid?.verify_hostnames === false) {
```

Comment:
Do we need to handle if `this.config.openid?.verify_hostnames` is undefined?

Reply:
not necessary, because `this.config.openid` will definitely be an object because this is OpenID auth, and verify_hostnames has default value of `true`